server/mailet/integration-testing/src/test/java/org/apache/james/mailets/utils/IMAPMessageReader.java

```diff
@@ -60,6 +60,13 @@ public String readFirstMessageInInbox(String user, String password) throws IOExc
         return imapClient.getReplyString();
     }
 
+    public String readFirstMessageHeadersInInbox(String user, String password) throws IOException {
```

Comment:
Very close to readFirstMessageInInbox.

Please consider factorizing the two method with a new one : "readFirstMessageInInboxWithParameter"